docs/ha-considerations.md

```diff
@@ -317,27 +397,59 @@ metadata:
   namespace: kube-system
 spec:
   containers:
-  - command:
-    - /kube-vip
-    - start
-    - -c
-    - /vip.yaml
-    image: 'plndr/kube-vip:0.1.1'
+  - args:
+    - manager
+    env:
+    - name: vip_arp
+      value: "false"
+    - name: port
+      value: "6443"
+    - name: vip_interface
+      value: ens192
+    - name: vip_cidr
+      value: "32"
+    - name: cp_enable
+      value: "true"
+    - name: cp_namespace
+      value: kube-system
+    - name: vip_ddns
+      value: "false"
+    - name: bgp_enable
+      value: "true"
+    - name: bgp_routerid
+      value: 192.168.0.2
+    - name: bgp_as
+      value: "65000"
+    - name: bgp_peeraddress
+    - name: bgp_peerpass
+    - name: bgp_peeras
+      value: "65000"
+    - name: bgp_peers
+      value: 192.168.0.10:65000::false,192.168.0.11:65000::false
+    - name: vip_address
+      value: 192.168.0.40
+    image: ghcr.io/kube-vip/kube-vip:v0.4.0
+    imagePullPolicy: Always
```

Comment:
The imagePullPolicy is `IfNotPresent` by default. Should we change the default value?

Reply:
hmm, this is in the code of Kube-vip as it generates the manifests.. so the documentation is correct, but the default behaviour of Kube-vip manifest generation will need to change.